### DISCUSSION TITLE
What is behind/How to interpret "Estimated loss"?

### DISCUSSION THREAD

**Discussion by TigersWay** [wiresock/WireSockUI#25] [Q&A] [answered]
Hi,
Have been using WireSockUI for a few weeks with a great pleasure, I can now use my VPNs in a smart way 😄 

But I just discovered the `Estimated loss` field which during a specific connection to a specific VPN gave me ~=26%, which seems to me a **pretty huge number**!
Could you share a little help on where to go from there? How to diagnose and find the culprit(s)?

Thanks

**Comment by wiresock** [comment, marked as answer]
Hi there,

I'm glad to hear you've been enjoying WireSockUI! 😄

A 26% packet loss is indeed significant. One common issue with VPNs, especially in relation to packet loss, is the MTU (Maximum Transmission Unit) size. I recommend trying to reduce the MTU to 1280 for the VPN connection experiencing the high packet drop rate and see if it improves the situation. Adjusting the MTU can sometimes mitigate these types of issues.

Best wishes.

**Reply by TigersWay** [reply]
I can quickly confirm that "playing" with the MTU (I didn't even knew we could) may change a lot of things:
- less loss of course (went down to ~=3%),
- and a very different download speed (More than doubled)!

Just wondering how to adjust it, but I'll look around.

Thanks a lot again!
